Yoast/wordpress-seo
# Hide the meta keywords section when in the readability tab.

## Summary

Hide the "Meta keywords" meta box section when switching to the "Readability" tab.

## Test instructions

This PR can be tested by following these steps:

- build the JS
- be sure the "Titles & Metas > Other > Use meta keywords tag?" setting is set to "Enabled"
- edit a post, notice the additional section "Meta keyword" is displayed within the meta box
- switch to the meta box "Readability" tab
- the "Meta keyword" section should disappear

Fixes #6393
